### ISSUE TITLE
Add styles

### ISSUE THREAD

**Issue by Rubenacp** [DrCyanide/cyanic-sd-krita#6]
I just wanted to comment that this extension is incredible. The auto-sd-krita is also very good when it works, but Controlnet is a big leap. Soon I will make some video tutorials to show its potential. 

**One question, how can I add styles?** 

Thank you

**Comment by DrCyanide** [comment]
Thanks, glad you're enjoying it so far! Things are still changing fairly frequently right now, so a tutorial video might be a little dated in a few weeks, but feel free to share it. 

As for adding styles from the extension... well....
Part of why auto-sd-krita broke is they didn't use the API that A1111 provides, but tried to tap directly into the internal functions. Whenever A1111 felt they needed to change something internally it would break. To avoid having that happen again I've been using the API. The API lets me _get_ the styles without issues, but there's no function to _set_ them.

TL;DR: Until the A1111 API updates, to add new styles you have to go to the Web UI.

**Comment by DrCyanide** [comment]
Created a feature request in A1111 for this.
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/13669

**Comment by Rubenacp** [comment]
Great, so it looks like this will be an extension that works long term, cool

thank you!
